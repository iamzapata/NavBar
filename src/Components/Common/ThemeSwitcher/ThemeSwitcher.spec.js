import React from "react"
import { render, cleanup } from "react-testing-library"
import { Provider } from "react-redux"
import { ThemeSwitcher } from "./index"

const createProps = () => ({
  theme: "Pinterest",
  switchTheme: jest.fn()
})

describe("<ThemeSwitcher />", () => {
  afterEach(cleanup)
  it("Should render self", () => {
    const { container } = render(<ThemeSwitcher {...createProps()} />)
    expect(container.firstChild).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })
})
