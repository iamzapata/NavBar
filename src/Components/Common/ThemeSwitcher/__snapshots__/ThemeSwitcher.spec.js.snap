// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemeSwitcher /> Should render self 1`] = `
<div
  class="ThemeSwitcher field Pinterest"
>
  <input
    checked=""
    class="switch is-rounded"
    id="themeSwitcherButton"
    type="checkbox"
  />
  <label
    for="themeSwitcherButton"
  />
</div>
`;
